server/src/telemetry.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2023 - 2025 Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use std::{str::FromStr, time::Duration};
+
+use http::{uri::PathAndQuery, HeaderMap, HeaderValue, Uri};
+use restate_core::{cancellation_watcher, TaskCenter, TaskKind};
+use restate_service_client::HttpClient;
+use restate_types::config::CommonOptions;
+use tokio::time::Instant;
+use tracing::debug;
+
+use crate::build_info::{
+    RESTATE_SERVER_DEBUG, RESTATE_SERVER_TARGET_TRIPLE, RESTATE_SERVER_VERSION,
+};
+
+static TELEMETRY_URI: &str = "https://restate.gateway.scarf.sh/restate-server/";
```

Comment:
Do we have a company account set up?

Reply:
we have an org, i need to upgrade the plan